src/polkadot_rpc_wrapper.py

```diff
@@ -73,7 +73,7 @@ def get_system_peers(self) -> Tuple[list, bool]:
         peer_list = response_json['result']
         return peer_list, True
 
-    def has_session_key(self, session_key):
+    def has_session_key(self, session_key) -> bool:
```

Comment:
It does not return a bool right?

Reply:
Ah you are probably right
https://polkadot.js.org/docs/kusama/rpc/#hassessionkeyssessionkeys-bytes-bool